internal/kibana/reconciler.go

```diff
@@ -421,11 +424,18 @@ func getImage() string {
 	return utils.LookupEnvWithDefault("KIBANA_IMAGE", kibanaDefaultImage)
 }
 
-func getProxyImage() string {
-	return utils.LookupEnvWithDefault("PROXY_IMAGE", kibanaProxyDefaultImage)
+func (clusterRequest *KibanaRequest) getProxyImage(ctx context.Context, c client.Client) string {
+	// Get the existing imagestream
+	key := client.ObjectKey{Name: oauthProxyImageName, Namespace: oauthProxyNamespace}
+	is, err := img.Get(ctx, c, key)
+	if err != nil {
+		clusterRequest.log.Error(err, "Failed to get oauth image stream")
+		return ""
+	}
+	return is.Status.Tags[0].Items[0].DockerImageReference
```

Comment:
I think `Status.Tags` might be empty early on before the ImageStream has been imported?  Feels like you should have some length guards in here to avoid panics, just in case.